### PR TITLE
Remove tabler import from glpi custom css

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

Continuation of #16175. The last PR focused on removing tabler/bootstrap imports from palette/theme CSS stylesheets and separating them from the rest of the GLPI custom css. This PR works to remove tabler/bootstrap imports from the rest of the GLPI custom css.

### CODE REVIEW

**Comment by cconard96** [comment]
Execution time of bin/console build:compile_scss for me:
- main branch before the last PR: 6 minutes
- main branch currently: 30 seconds
- After this PR: 18 seconds

For me, CSS compilation is now 20x faster.

**Comment by orthagh** [comment]
And also important one, a change in one of our css make an almost instant compilation where before it took 5-6s on my local instance.

**Review by orthagh — APPROVED** [review]
Good job ! 👍🏽

**Comment by orthagh** [comment]
A quick issue probably present since the previous pr, the filters in dashboard displays a box-shadow inside their outer border.
It's probably visible only in light palettes
image

**Review by trasher — APPROVED** [review]

File: css/glpi.scss [hunk 1]
```diff
@@ -34,6 +34,12 @@
...
-@import "includes/logos";
 @import "includes/includes";
+
+html:not([data-glpi-theme-dark="1"]) {
+    @import "../public/lib/tinymce/skins/ui/oxide/skin";
+}
+
+html[data-glpi-theme-dark="1"] {
+    @import "../public/lib/tinymce/skins/ui/oxide-dark/skin";
+}
```
Comment (cedric-anne) [inline css/glpi.scss:45]: You should add a rule in the webpack config to copy these files into `css/lib` (the same way it is done for tabler/bootstrap scss files).
```js
    {
        package: 'tinymce',
        from: 'skins/ui/oxide-*/skin.css',
        to: scssOutputPath,
    },
```

And then use the `~` prefix to import them.
```suggestion
html:not([data-glpi-theme-dark="1"]) {
    @import "~tinymce/skins/ui/oxide/skin";
}

html[data-glpi-theme-dark="1"] {
    @import "~tinymce/skins/ui/oxide-dark/skin";
}
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

File: .webpack.config.js [hunk 2]
```diff
@@ -172,6 +172,11 @@ var filesToCopy = [
         from: '{fonts/*,tabler-icons.scss}',
         to: scssOutputPath,
     },
+    // {
```
Comment (trasher) [inline .webpack.config.js:175]: Should not be commented or not be added :)

Comment (orthagh) [reply .webpack.config.js]: yeah yeah, I commited to discuss it with Cédric

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**